### DISCUSSION TITLE
Sensors description

### DISCUSSION THREAD

**Discussion by nakoukal** [mletenay/home-assistant-goodwe-inverter#97] [Ideas]
Hi , thanks for this integration , i would like to ask to description of sensors. Most of sensors are self-explanatory but some are confused. For example : goodwe_e_day_imp (Today energy import) from name i suppose That this sensor shows my consumption from grid but it doesn't . My value shows this sensor goodwe_e_load_day.  Another example : goodwe_ups_load ? Is somewhere described what these sensors measure?

**Comment by mletenay** [comment]
No, not really.
The sensors (and their names) were collected from various sources found over the net and from reverse engineering the protocol.
As you said, some are obvious, some not.

**Comment by jorgestima** [comment]
Is it possible to clarify which power indicator corresponds to the effective load of house consumption? Initially I was assuming that "House Consumption" was what I was looking for but then I've noticed some strange things like negative power during night, which does not make any sense because it would mean that I would have some device sending power to grid, which is impossible. There are other power indicators such as "On-Grid Power", "Plant Power" or "Total Power" so I'm very confused. 

Any help on this?
